oshmem/mca/spml/ucx/spml_ucx.c

```diff
@@ -566,12 +566,33 @@ void mca_spml_ucx_rmkey_free(sshmem_mkey_t *mkey, int pe)
 {
     spml_ucx_mkey_t   *ucx_mkey;
     uint32_t segno;
-    int rc;
+    int i,rc;
 
     if (!mkey->spml_context) {
         return;
     }
-    segno = memheap_find_segnum(mkey->va_base);
```

Comment:
1. what this function returns if the va_base isn't found?
2. Can we modify it to return MEMHEAP_SEG_INVALID if it's not found?
Basically, should we use the code introduced here to implement `memheap_find_segnum()`?